modules/core/src/main/scala/org/scalasteward/core/repoconfig/RepoConfigAlg.scala

```diff
@@ -16,34 +16,49 @@
 
 package org.scalasteward.core.repoconfig
 
+import better.files.File
 import cats.data.OptionT
 import cats.implicits._
 import io.chrisdavenport.log4cats.Logger
 import io.circe.config.parser
+import org.scalasteward.core.application.Config
 import org.scalasteward.core.data.Update
 import org.scalasteward.core.io.{FileAlg, WorkspaceAlg}
 import org.scalasteward.core.repoconfig.RepoConfigAlg._
 import org.scalasteward.core.util.MonadThrowable
 import org.scalasteward.core.vcs.data.Repo
 
 final class RepoConfigAlg[F[_]](implicit
+    config: Config,
     fileAlg: FileAlg[F],
     logger: Logger[F],
     workspaceAlg: WorkspaceAlg[F],
     F: MonadThrowable[F]
 ) {
   def readRepoConfigOrDefault(repo: Repo): F[RepoConfig] =
-    readRepoConfig(repo).map(_.getOrElse(RepoConfig.default))
+    readRepoConfig(repo).flatMap { config =>
+      config.map(F.pure).getOrElse(defaultRepoConfig())
+    }
+
+  /**
+    * Default configuration will try to read file specified in config.reposDefaultConfig first; if not found - fallback to empty configuration.
+    * Note it's not lazy since we want to get config updates if you decide to change config in the middle of the process
```

Comment:
Could you please break lines after 80 characters?

Reply:
No problem
Do you want to change https://github.com/fthomas/scala-steward/blob/master/.scalafmt.conf#L5 to be 80?

Reply:
I thought this setting has no effect on Scaladocs?
We shouldn't change that setting in this PR.

Reply:
it has, otherwise I'd have it formatted by `fmt`